Cannot record a new password with console error : Empty result from get_database_groups

<!--- Provide a general summary of the issue in the title above -->
## Expected Behavior
I write my password, the extension is asking me to click the button to add it.

## Current Behavior
Nothing append, the banner stay on the screen.

## Debug info
<!--- Please copy/paste the browser's JavaScript console messages and errors here -->
On the console log :
`banner.js:141 Error: Empty result from get_database_groups`

KeePassXC - 2.3.4
KeePassXC-Browser - 1.7.1
Operating system: Linux x86_64 (Debian stretch with backport of KeePassXC)
Browser: Chrome/Chromium 85.0.4183.83 (Brave)

--- a/keepassxc-browser/content/banner.js
+++ b/keepassxc-browser/content/banner.js
@@ -136,18 +136,22 @@ kpxcBanner.create = async function(credentials = {}) {
 };
 
 kpxcBanner.saveNewCredentials = async function(credentials = {}) {
+    const saveToDefaultGroup = async function(creds) {
+        const args = [ creds.username, creds.password, creds.url ];
+        const res = await sendMessage('add_credentials', args);
+        kpxcBanner.verifyResult(res);
+    };
+
     const result = await sendMessage('get_database_groups');
     if (!result || !result.groups) {
         console.log('Error: Empty result from get_database_groups');
+        await saveToDefaultGroup(credentials);
         return;
     }
 
     if (!result.defaultGroupAlwaysAsk) {
         if (result.defaultGroup === '' || result.defaultGroup === DEFAULT_BROWSER_GROUP) {
-            // Default group is used
-            const args = [ credentials.username, credentials.password, credentials.url ];
-            const res = await sendMessage('add_credentials', args);
-            kpxcBanner.verifyResult(res);
+            await saveToDefaultGroup(credentials);
             return;
         } else {
             // A specified group is used

